webdriver-spec.html

```diff
@@ -4624,636 +4646,562 @@
 </section> <!-- /Delete All Cookies -->
 </section> <!-- /Cookies -->
 
-<section>
-<h2>Interactions</h2>
-
-<p class=issue><strong>Warning:</strong>
- This section has not yet been redefined to match the <a>routing requests</a> model,
- and uses old concepts and definitions.
- Please do not rely on it yet.
+<section><!-- Actions -->
+  <h2>Interactions</h2>
+  The Actions API gives you a mechanism to do atomic interactions by sending over
+  a JSON object that can then be used to <a>dispatch commands</a>.
+  The commands are then <a data-lt=command-processed>processed</a> by the user agent,
+  allowing complex interactions to be created.
```

Comment:
This feels a little unspecific.  I think a nicer phrasing would be that it allows you to _pipeline interaction actions to be performed atomically_ on the remote end, or something along those lines.